karma.conf.js

```diff
@@ -25,6 +25,7 @@ module.exports = config => {
     colors: true,
     frameworks: ['mocha'],
     files: [
+      'node_modules/regenerator-runtime/runtime.js',
```

Comment:
what is for?

Reply:
for the async await syntax to work (since the its test runner run code from playkit which contains async await)